cnxdb/migrations/20170918131724_add_featured_books.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    cursor.execute("""\
+CREATE TABLE featured_books (
+    "uuid" UUID NOT NULL,
+    "major_version" INT,
+    "minor_version" INT
+);""")
+    cursor.execute("""\
+INSERT INTO featured_books (uuid, major_version, minor_version)
+    SELECT uuid, major_version, minor_version FROM modules
+        JOIN moduletags ON moduletags.module_ident = modules.module_ident
+        WHERE moduletags.tagid=9;
```

Comment:
We need the ones tagged `openstax featured` as well, so instead, use the schema `featured`, and keep the tagid in the table as well.

Reply:
By "use the schema `featured`" I mean `tags.schema = 'featured'`